js/5etools-main.js

```diff
@@ -3860,482 +3860,8 @@ To restore this functionality, press the "Bind Drag-n-Drop" button.<br>
-				let tokenList;
-
-				const dataStack = (await Promise.all(toLoad.map(url => DataUtil.loadJSON(url)))).flat();
-
-				$lst.empty();
-				let toShow = [];
-
-				const seen = {};
-				await Promise.all(dataStack.map(async d => {
-					const toAdd = d.monster.filter(m => {
-						const out = !(seen[m.source] && seen[m.source].has(m.name));
-						if (!seen[m.source]) seen[m.source] = new Set();
-						seen[m.source].add(m.name);
-						return out;
-					});
-
-					toShow = toShow.concat(toAdd);
-				}));
-
-				toShow = toShow.sort((a, b) => SortUtil.ascSort(a.name, b.name));
-
-				let tmp = "";
-				toShow.forEach((m, i) => {
-					m.__pType = Parser.monTypeToFullObj(m.type).asText;
-
-					tmp += `
-								<label class="import-cb-label" data-listid="${i}">
-									<input type="checkbox">
-									<span class="name col-4">${m.name}</span>
-									<span class="type col-4">TYP[${m.__pType.uppercaseFirst()}]</span>
-									<span class="cr col-2">${m.cr === undefined ? "CR[Unknown]" : `CR[${(m.cr.cr || m.cr)}]`}</span>
-									<span title="${Parser.sourceJsonToFull(m.source)}" class="source">SRC[${Parser.sourceJsonToAbv(m.source)}]</span>
-								</label>
-							`;
-				});
-				$lst.html(tmp);
-				tmp = null;
-
-				tokenList = new List("shapeshiftbuild-list", {
-					valueNames: ["name", "type", "cr", "source"],
-				});
-
-				d20plus.importer.addListFilter($fltr, toShow, tokenList, d20plus.monsters._listIndexConverter);
-
-				$win.find(`button`).on("click", () => {
-					function getSizeInTiles (size) {
-						switch (size) {
-							case SZ_TINY:
-								return 0.5;
-							case SZ_SMALL:
-							case SZ_MEDIUM:
-								return 1;
-							case SZ_LARGE:
-								return 2;
-							case SZ_HUGE:
-								return 3;
-							case SZ_GARGANTUAN:
-								return 4;
-							case SZ_COLOSSAL:
-								return 5;
-						}
-					}
-
-					if (tokenList) {
-						$("a.ui-tabs-anchor[href='#deckstables']").trigger("click");
-
-						const sel = tokenList.items
-							.filter(it => $(it.elm).find(`input`).prop("checked"))
-							.map(it => toShow[$(it.elm).attr("data-listid")]);
-
-						const id = d20.Campaign.rollabletables.create().id;
-						const table = d20.Campaign.rollabletables.get(id);
-						table.set("name", $(`#shapeshift-name`).val().trim() || "Shapeshifter");
-						table.save();
-						sel.forEach(m => {
-							const item = table.tableitems.create();
-							item.set("name", m.name);
-							// encode size info into the URL, which will get baked into the token
-							const avatar = m.tokenUrl || `${IMG_URL}${Parser.sourceJsonToAbv(m.source)}/${m.name.replace(/"/g, "")}.png?roll20_token_size=${getSizeInTiles(m.size)}`;
-							item.set("avatar", avatar);
-							item.set("token_size", getSizeInTiles(m.size));
-							item.save();
-						});
-						table.save();
-						d20.rollabletables.refreshTablesList();
-						alert("Created table!")
-					}
-				});
-			},
-		},
-		{
-			name: "Pauper's Character Vault",
```

Comment:
What's the justification here? It already exists in VTTES..?
Please take the time to explain the _why_ when making PRs like this, either in the commit message (probably preferred in this case) or in a comment/the description of the PR

Reply:
Apologies for not stating, thought the trello counted. The pauper's vault is a duplicate feature of what VTTES offers.

Reply:
well there's the problem, I don't read the Trello 🙃 
Ideally, the PR should have a "key points" summary, and a link to the Trello where appropriate